src/jekyll/zeppelin.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: global
+displayTitle: Apache Zeppelin running with Spark on Kubernetes
+title: Apache Zeppelin running with Spark on Kubernetes
+description: User Documentation for Apache Zeppelin running with Spark on Kubernetes
+---
+
+This page a ongoing effort to describe how to run Apache Zeppelin with Spark on Kubernetes
+
+> At the time being, the needed code is not integrated in the `master` branches of `apache-zeppelin` nor the `apache-spark-on-k8s/spark` repositories.
+> You are welcome to already ty it out and send any feedback and question.
+
+Firs things firs, you have to choose the following modes in which you will run Zeppelin with Spark on Kubernetes:
+
++ The `Kubernetes modes`: Can be `in-cluster` (within a Pod) or `out-cluster` (from outside the Kubernetes cluster).
++ The `Spark deployment modes`: Can be `client` or `cluster`.
+
+Only three combinations of these options are supported:
+
+1. `in-cluster` with `spark-client` mode.
+2. `in-cluster` with `spark-cluster` mode.
+3. `out-cluster` with `spark-cluster` mode.
+
+For now, to be able to test these combinations, you need to build specific branches (see hereafter) or to use third-party Helm charts or Docker images. The needed branches and related PR are listed here:
+
+1. Spark-k8s driven branch: In-cluster client mode [see pull request #456](https://github.com/apache-spark-on-k8s/spark/pull/456)
+2. Apache Zeppeoin driven branch: Add support to run Spark interpreter on a Kubernetes cluster [see pull request #2637](https://github.com/apache/zeppelin/pull/2637)
```

Comment:
Zeppelin?
what is `driven branch`?

Reply:
Just wanted to point where this branch resides... I have remove that to avoid confusion.